tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -56,89 +56,62 @@ public function tearDown()
         parent::tearDown();
     }
 
-    /**
-     * Kill all sessions on the cluster.
-     *
-     * This will clean up any open transactions that may remain from a
-     * previously failed test. For sharded clusters, this command will be run
-     * on all mongos nodes.
-     */
-    private static function killAllSessions()
-    {
-        $manager = new Manager(static::getUri());
-        $primary = $manager->selectServer(new ReadPreference('primary'));
-
-        $servers = ($primary->getType() === Server::TYPE_MONGOS)
-            ? $manager->getServers()
-            : [$primary];
-
-        foreach ($servers as $server) {
-            try {
-                // Skip servers that do not support sessions
-                if (!isset($server->getInfo()['logicalSessionTimeoutMinutes'])) {
-                    continue;
-                }
-                $server->executeCommand('admin', new Command(['killAllSessions' => []]));
-            } catch (ServerException $e) {
-                // Interrupted error is safe to ignore (see: SERVER-38335)
-                if ($e->getCode() != self::INTERRUPTED) {
-                    throw $e;
-                }
-            }
-        }
-    }
-
     /**
      * Assert that the expected and actual command documents match.
      *
-     * Note: this method may modify the $expectedCommand object.
+     * Note: this method may modify the $expected object.
      *
-     * @param stdClass $expectedCommand Expected command document
-     * @param stdClass $actualCommand   Actual command document
+     * @param stdClass $expected Expected command document
+     * @param stdClass $actual   Actual command document
      */
-    public function assertSameCommand(stdClass $expectedCommand, stdClass $actualCommand)
+    public static function assertCommandMatches(stdClass $expected, stdClass $actual)
```

Comment:
PHPUnit tends to prefer `static` methods for assertions when possible. This wasn't necessary for our own needs, but I didn't see the harm in following convention. I haven't given this much thought in the past because I was seldom digging through PHPUnit's source; however, this came up while I was crafting our custom Constraint class.